decred_wallet/view_controller/SendViewController.swift

```diff
@@ -495,7 +495,11 @@ class SendViewController: UIViewController, UITextFieldDelegate,UITextPasteDeleg
             }
             self.showDefaultAccount()
             }
-            self.openLink(urlString: "https://testnet.dcrdata.org/tx/" + hashe! )
+            if(UserDefaults.standard.bool(forKey: "walletScanning")){
```

Comment:
Why does this check `walletScanning`? Isn't it supposed to check if app is on testnet mode?